lib/awful/hotkeys_popup/widget.lua

```diff
@@ -44,22 +55,71 @@ widget.hide_without_description = true
 widget.merge_duplicates = true
 
 
+--- Widget background color.
+-- @beautiful beautiful.hotkeys_bg
```

Comment:
for some reason those beautiful docstrings are displayed in apidoc for the hotkeys_popup.widget but not present in 06-appearance

could it be because they're missing `@param`?